project/assets/login.less

```diff
@@ -0,0 +1,90 @@
+.body-structure {
+    width: 100%;
+}
+
+.body-skin {
+    background-color: rgb(230, 230, 230);
+}
+
+header {
+    background: url("images/feedback-banner.jpg") center center no-repeat;
+    opacity: 0.8;
+    height: 600px;
+    width: 100%;
+
+    .grid {
+        height: 100%;
+    }
+}
+
+.login-section-1 {
+    padding: 70px;
+}
+
+.login-section-1-skin {
+    background-color: rgb(255, 255, 255);
+}
+
+.login-section-text {
+    padding: 70px;
+}
+
+.login-section-2 {
+    padding: 30px;
+}
+
+.login-form {
+    align-self: center;
+    padding-bottom: 30px;
+    margin-left: 20%;
+    width: 25%;
+    height: 65%;
+}
+
+.login-form-skin {
+    background-color: rgb(255, 255, 255);
+}
+
+.login-form-items {
+    padding: 25px 25px 5px;
+}
+
+.login-form-header-skin {
+    background-color: rgb(51, 153, 255);
```

Comment:
This exact combination of color, font family, background color it's also duplicated in other file(new-evaluation.less). To increase reusability break down this class, may around these lines:
```css
.normal-font {
font-family: roboto;
font-size: 20px;
}
.bold {
font-weight: bold;
}
.text-title { // not sure about this name, find a better one
color: ...
background-color: ...
}
```
Given these classes remove any duplications of these properties from everywhere. Also look for other possibilities of reducing duplication